R/event.R

```diff
@@ -36,11 +36,12 @@ dyEvent <- function(dygraph,
   
   # create event
   event <- list()
-  event$date <- asISO8601Time(date)
-  event$label <- label
+  event$pos = asISO8601Time(date)
+  if (!missing(label) && !is.null(label)) event$label <- label
```

Comment:
Why this treatment rather than just having label = NULL as the default argument value?